src/table.js

```diff
@@ -1322,6 +1322,72 @@ Table.prototype.sampleRowKeysStream = function(gaxOptions) {
   ]);
 };
 
+/**
+ * Returns true in callback if consitency check is true for
+ * generated consistency token.
+ * In-case consistency check returns false, retrial is done in interval
+ * of 5 seconds till 10 minutes, after that it returns false.
+ *
+ * @param {function(?Error, ?boolean)} [callback]
```

Comment:
The other methods break down the components of the response, e.g. from `sampleRowKeys`:

```
 * @param {function} [callback] The callback function.
 * @param {?error} callback.err An error returned while making this request.
 * @param {object[]} callback.keys The list of keys.
```